changelog.d/9426.misc

```diff
@@ -0,0 +1 @@
+Calling the 'parse_duration' function so that configs can be specified in terms of number of seconds/hours/etc/.
```

Comment:
We want to word this in terms of how a Synapse administrator might care, so maybe:

```suggestion
The `ui_auth.session_timeout` configuration setting be specified in terms of number of seconds/minutes/etc/.
```

You can also add `Contributed by <your name>.` to the end if you'd like!